databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -134,10 +153,10 @@ def shared_data_object(name):
 @click.option('--owner', default=None, required=False,
               help='Owner of the share.')
 @click.option('--add-table', default=None, multiple=True,
-              metavar='NAME',
+              metavar='NAME', hidden=True,
               help='Full name of table to add to share (can be specified multiple times).')
 @click.option('--remove-table', default=None, multiple=True,
-              metavar='NAME',
+              metavar='NAME', hidden=True,
```

Comment:
~Why hide these?~

Can you add a comment why they are hidden? I suspect you'd like to encourage folks to use the `update-table` subcommand instead of this one.

Reply:
You're right!

<img width="798" alt="image" src="https://user-images.githubusercontent.com/113815940/197585523-0ca06d3e-9a38-42b2-b374-da8245193d30.png">